### ISSUE TITLE
Any idea for regression

### ISSUE THREAD

**Issue by KP-Zhang** [philipperemy/tensorflow-phased-lstm#5]
Hi, nice work.
I am wondering if there is any variant of phased lstm for regression problem.
Could you offer me any advice?

**Comment by philipperemy** [comment]
Thanks @DylanCheung ! Sure!

Replace the `10` by `1` here:
```
y_ = tf.placeholder('float32', [batch_size, 1])
[...]
fc0_w = create_weight_variable('fc0_w', [hidden_size, 1])
fc0_b = create_bias_variable('fc0_b', [1])
```

And replace this:
```
cross_entropy = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits=y, labels=y_))
grad_update = tf.train.AdamOptimizer(learning_rate).minimize(cross_entropy)
correct_prediction = tf.equal(tf.argmax(y_, 1), tf.argmax(y, 1))
accuracy = tf.reduce_mean(tf.cast(correct_prediction, tf.float32))
```

by
```
mse = tf.reduce_mean(tf.square(tf.sub(train_out, output)))
grad_update = tf.train.AdamOptimizer(learning_rate).minimize(mse)
```
`accuracy` does not make sense for a regression problem.

I haven't tested this code. Let me know if this can kickstart you!

**Comment by DylanCheung** [comment]
@philipperemy Thank you for your advice. It is very helpful.
I have a bunch of cyclical data sampling every 15 minutes, 96 points per day for two month. I employed t-2, t-1 and t to predict t+1 and got some results.
Now, I am using t-2, t-1, t combining with t-3*96+1, t-2*96+1, t-96+1 to predict t+1 due to  the cyclical feature of my data. For example I want to predicted Y at 10:00 PM 05/16/2017, my X data should be 
09:15 PM 05/16/2017    10:00 PM 05/13/2017
09:30 PM 05/16/2017    10:00 PM 05/14/2017
09:45 PM 05/16/2017    10:00 PM 05/15/2017   --> Y (10:00 PM 05/16/2017)
(I hope I described my thought clearly.)
After getting involved 10:00 PM 05/13/2017, 10:00 PM 05/14/2017, 10:00 PM 05/15/2017,  I receive a better result.
Inspired by your work, I'd like to use phased lstm to handle my regression prediction for time series data. I am not confidently say I understand phased lstm very well.
From your experience, Is phased lstm workable for my data? How could I construct my data structure if I use phased lstm?
Again, thank you for your wonderful work.
